src/customize-pop/accessibility-seeing.md

```diff
@@ -0,0 +1,187 @@
+    <img src="/images/accessibility-settings/contrast-mode-off.png" width="733px" height="456px">
+</a>
+
+### High Contrast Mode On
+
+**Click image to enlarge**
+
+<a href="/images/accessibility-settings/contrast-mode-on.png" target="blank">
+    <img src="/images/accessibility-settings/contrast-mode-on.png" width="733px" height="456px">
+</a>
+
+## Large Text
+
+Enabling `Large Text` increases the size of text across system windows and supported applications.
+
+### Large Text Disabled
+
+**Click image to enlarge**
+
+<a href="/images/accessibility-settings/before-lt.png" target="blank">
+    <img src="/images/accessibility-settings/before-lt.png" width="733px" height="456px">
+</a>
+
+### Large Text Enabled
+
+**Click image to enlarge**
+
+<a href="/images/accessibility-settings/after-lt.png" target="blank">
+    <img src="/images/accessibility-settings/after-lt.png" width="733px" height="456px">
+</a>
+
+## Enable or Disable Animations
+
+Animations appear when minimizing and maximizing windows. Disabling animations may improve performance on systems with low video resources.
+
+### Animations Enabled
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/animations-enabled.webm" />
+</video>
+
+### Animations Disabled
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/animations-disabled.webm" />
+</video>
+
+## Cursor Size
+
+`Cursor Size` allows users to choose between five cursor sizes.
+
+![Cursor Size](/images/accessibility-settings/cursor-size.png)
+
+## Zoom
+
+The `Zoom` setting designates all or a part of the screen to display magnified content. This feature can be further customized to render the zoomed area specific portions of the screen, increase or decrease the zoom magnification, add a crosshair, and add color effects to the zoomed area.
+
+**Click image to enlarge**
+
+<a href="/images/accessibility-settings/zoom-options.png" target="blank">
+    <img src="/images/accessibility-settings/zoom-options.png" width="733px" height="456px">
+</a>
+
+### Magnification
+
+This option determines the magnification of the zoom effect. Values range from 1-20 and correspond to percentages; a value of 4.0 applies a magnification of 400%.
+
+![Magnification](/images/accessibility-settings/magnification.png)
+
+### Follow Mouse Cursor & Screen Part
+
+If `Full Screen` is selected under `Screen part` and the `Follow mouse cursor` radio button is selected, the magnification affect is rendered to the entire display area.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/full-zoom.webm" />
+</video>
+
+### Follow Mouse Cursor & Half Screen Zoom
+
+Select a portion of the screen to display the rendered zoom effect. If `Follow mouse cursor` remains selected, the `Top Half` and `Bottom Half` options will render the zoom effect to an unfixed narrow band of the screen. The `Left Half` and `Right Half` options will render the zoom effect to an unfixed rectangular portion of the screen.
```

Comment:
"narrow band" and "rectangular portion" are a little vague and interchangeable; perhaps "horizontal band" and "vertical band" or something like that could be used instead.